docs/development.md

```diff
@@ -315,5 +315,6 @@ There are several files used to configure various aspects of the system.
 * `web.xml`: Contains the web server configuration, e.g servlets to run, mapping from URLs to servlets/JSPs, security constraints, etc.
 * `cron.xml`: Contains the cron jobs specification.
 * `queue.xml`: Contains the task queues configuration.
+* `datastore-indexes.xml`: Contains the Datastore indexes configuration.
```

Comment:
Should we add a `datastore-indexes-auto.xml` description here as well? See below.